css/index.css

```diff
@@ -1,4 +1,12 @@
 /* stylelint-disable max-empty-lines */
 /* stylelint-disable no-empty-source */
```

Comment:
- [ ][ optional] it would be better to remove the Stylelint disable cause the linter ensures writing a clean code so please try to do so